### DISCUSSION TITLE
Xvg wallet iphone

### DISCUSSION THREAD

**Discussion by RoelBodt** [community/community#84069] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, 

I’ve tried to open my xvg wallet on my iPhone. But it will not open. I get the message tor didn’t connect properly. How can I fix this ?

Kind regards Roel

**Comment by Akash1134** [comment]
Hi @RoelBodt ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
